scripts/build-agent-image

```diff
@@ -36,7 +36,16 @@ cp ./misc/certs/host-certs.crt rootfs/etc/ssl/certs/ca-certificates.crt
 
 # add pause container
 mkdir -p rootfs/images/
-cp ./misc/pause-container/amazon-ecs-pause.tar rootfs/images/amazon-ecs-pause.tar
+# check for precompiled pause image tar -- if one doesn't exist,
+# you need to run make dockerfree-pause
+PAUSE_TAR_FILE="${ROOT}/misc/pause-container/pause-image-tar-files/amazon-ecs-pause.tar"
```

Comment:
Where do we write to this path? `build-pause` seems to write to `amazon-ecs-pause-${GOARCH}.tar`?

Reply:
We don't specifically _write_ to this path, we have the prebuilt pause container stored at this path, so we just use that to get past the `glibc` dependency.